distribution/docs/src/main/resources/_contents/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,75 @@
+==== Metacard Ingest Network Plugin
```

Comment:
I think it's a reasonable expectation that programmers will think the file name is the same as the module; that's at least according to the principle of least astonishment. Would be a shame if it didn't appear on a search for `metacardingest` literally.